client/src/components/admin/student/AddStudent.jsx

```diff
@@ -1,66 +1,186 @@
+      [id]: value,
+      showSuccessMessage: false
+    });
+  };
+
+  handleSelectionChange = (e, { id, value }) => this.setState({ [id]: value });
+
+  addStudent = () => {
+    const { cohorts } = this.props;
+    const {
+      enrollmentId,
+      firstName,
+      lastName,
+      githubHandle,
+      zoomName,
+      cohort,
+      enrollmentStatus
+    } = this.state;
+
+    const selectedCohort = cohorts.find(currentCohort => currentCohort.name === cohort);
+    if (selectedCohort && selectedCohort.id) {
+      const { id } = selectedCohort;
+      axios
+        .post(
+          `${GHOSTBUSTER_BASE_URL}/ghostbuster/students?enrollmentId=${enrollmentId}
+          &firstName=${firstName}&lastName=${lastName}&github=${githubHandle}
+          &zoomName=${zoomName}&cohortId=${id}&status=${enrollmentStatus}`
+        )
+        .then(response => {
+          if (response.data.student) {
+            this.setState({
+              enrollmentId: '',
+              firstName: '',
+              lastName: '',
+              githubHandle: '',
+              zoomName: '',
+              cohort: '',
+              enrollmentStatus: enrollmentStatusList[3].value,
+              showSuccessMessage: true
+            });
+          }
+        });
+    }
+  };
+
   render() {
     const { cohorts } = this.props;
     const cohortsList = cohorts.map(cohort => ({
       key: cohort.name,
       text: cohort.name,
       value: cohort.name
     }));
+    const {
+      enrollmentId,
+      firstName,
+      lastName,
+      githubHandle,
+      zoomName,
+      cohort,
+      enrollmentStatus,
+      showSuccessMessage
+    } = this.state;
     return (
       <React.Fragment>
         <Header as="h1" textAlign="center">
           Add Student
+          {showSuccessMessage ? (
+            <div>
+              <Message positive compact size="mini">
+                <Message.Header>Student added!</Message.Header>
+              </Message>
+            </div>
+          ) : (
+            <div />
+          )}
         </Header>
-        <Form>
-          <Form.Field>
-            <label htmlFor="firstName">
-              First Name
-              <input type="text" id="firstName" placeholder="First Name" />
-            </label>
-          </Form.Field>
-          <Form.Field>
-            <label htmlFor="lastName">
-              Last Name
-              <input type="text" id="lastName" placeholder="Last Name" />
-            </label>
-          </Form.Field>
-          <Form.Field>
-            <label htmlFor="githubHandle">
-              Github Handle
-              <input type="text" id="lastName" placeholder="Github Handle" />
-            </label>
-          </Form.Field>
-          <Form.Field>
-            <label htmlFor="zoomName">
-              Zoom Name
-              <input type="text" id="lastName" placeholder="Zoom Name" />
-            </label>
-          </Form.Field>
+        <Form onSubmit={this.addStudent}>
+          <Form.Field
```

Comment:
The User should not be providing the enrollment id for the new student.  When the addStudent function adds them, it will take the cohort specified and find the appropriate id for that cohort.  Using the new student record's id and the cohort id it has looked up, it should then create an enrollment record (which will auto-generate an enrollment id).

Reply:
removed the `Enrollment Id` field from the UI for now. Will make changes in the backend after the pr for the updated schema is merged